pathway-analysis/qusage_single_dataset.Rmd

```diff
@@ -97,16 +108,23 @@ not already.
 
 ```{r}
 # if the unzipped folder does not exist
-if (!dir.exists(file.path("data", "GSE75574"))) {
-  compressed_file <- file.path("data", "GSE75574.zip")
-  unzip(compressed_file, exdir = "data")
+if (!dir.exists(file.path(data_dir,
+                          "GSE75574" # Replace with the name of your file without the .zip extension
+                          ))) {
+  compressed_file <- file.path(data_dir,
+                               "GSE75574.zip" # Replace with the name of your zipped file
+                               )
+  unzip(compressed_file, exdir = data_dir)
 }
 ```
 
 ### Expression data
 
 ```{r}
-expression_file <- file.path("data", "GSE75574", "GSE75574.tsv")
+expression_file <- file.path(data_dir,
+                             "GSE75574", # Replace with name of your unzipped folder
```

Comment:
I would suggest just searching for `zip` so you make sure you catch all of these.